### ISSUE TITLE
Hi, could anyone guide me on implementing new loss function.Particulary loss functions which handle class imbalances,some of which are present in caffe and mxnet eg Infogain Loss and the MOON loss.

### ISSUE THREAD

**Issue by akset8** [fchollet/keras#5183]

**Comment by akset8** [comment]
References are : https://arxiv.org/abs/1603.07027
